src/components/buttons-container.js

```diff
@@ -0,0 +1,16 @@
+import PropTypes from 'prop-types';
+
+function ButtonsContainer(props) {
+  const { children } = props;
+  return (
+    <div className="buttons-container">
+      {children}
+    </div>
+  );
+}
+
+ButtonsContainer.propTypes = {
+  children: PropTypes.node.isRequired,
+};
+
+export default ButtonsContainer;
```

Comment:
- [ ] According to the project requirement you need to create a new file inside your components folder called `Calculator.js`  and this `Calculator.js` component should be **class-based**. Please fix it because as a developer it is very important to always follow the requirements because they establish a foundation for product vision, scope, cost, and schedule, and they ultimately must target finished product quality and performance. :+1:

Reply:
Done!